### PR TITLE
Create lab2 showcase progression on allthethings and add to bubble choice UI test

### PR DESCRIPTION
**Pull request by sanchitmalhotra126**
This creates a "Lab2 showcase" progression on allthethings as a place to test various cross-lab2 functionality. The immediate need for this was to add a UI test for bubble choice sublevels on lab2, which has been added to bubble_choice.feature. Also includes a related convenience fix to handle invalid sources in Weblab2.

Demo:
(note that the light mode footer for music lab showing up in this video is a [known issue](https://codedotorg.atlassian.net/browse/LABS-34), only manifesting when music lab is part of a progression with other light mode lab2 labs. This will not manifest on any production music lab levels).

https://github.com/code-dot-org/code-dot-org/assets/85528507/b6532583-a3b5-4200-8f68-307ee44dbbaf

## Testing story

Ran UI tests against localhost with saucelabs connect.

### CODE REVIEW

File: dashboard/test/ui/features/teacher_tools/level_types/bubble_choice.feature [hunk 1]
```diff
@@ -39,3 +39,68 @@ Feature: BubbleChoice
     When I click selector ".teacher-panel table td:contains(Alice)" once I see it to load a new page
     And I wait for 4 seconds
     Then I verify progress for the sublevel with selector ".uitest-bubble-choice:eq(0) .progress-bubble:first" is "perfect"
+
+  Scenario: Lab2 BubbleChoice progress
```
Comment (sanchitmalhotra126) [inline dashboard/test/ui/features/teacher_tools/level_types/bubble_choice.feature:43]: Tagged @code-dot-org/teacher-tools just as an FYI that I updated this UI test

File: apps/src/lab2/responseValidators.ts [hunk 2]
```diff
@@ -30,20 +30,20 @@ const BlocklySourceResponseValidator: ResponseValidator<
   return sourceValidatorHelper(response, blocklyValidator);
 };
 
-// Validator for Python sources.
-const PythonSourceResponseValidator: ResponseValidator<
+// Validator for CodeBridge sources.
+const CodeBridgeSourceResponseValidator: ResponseValidator<
```
Comment (sanchitmalhotra126) [inline apps/src/lab2/responseValidators.ts]: @molly-moen @thomasoniii made the validator updates here, let me know if something doesn't look right!

Comment (molly-moen) [reply apps/src/lab2/responseValidators.ts]: `Codebridge` should be all one word (instead of `CodeBridge`)

**Review by molly-moen — COMMENTED** [review]
One note on casing, otherwise looks good! Thanks for doing this!

File: apps/src/lab2/responseValidators.ts (hunk 2 above)
Comment (sanchitmalhotra126) [reply apps/src/lab2/responseValidators.ts]: Done 👍

**Review by molly-moen — APPROVED** [review]

**Pull request merged**